templates/server.properties.j2

```diff
@@ -5,8 +5,10 @@
 
 # The id of the broker. This must be set to a unique integer for each broker.
 # If left off one is auto-assigned
-{% if kafka.broker_id is defined %}
-broker.id={{ kafka.broker_id }}
+{% if kafka.aws_cluster_autodiscover.data is defined %}
```

Comment:
Unfortunately not, the parent list (kafka.aws_cluster_autodiscover.data) might not exist at all, in which case local VM setup would break completely on this line.